sips/sip-16/sip-016-token-metadata.md

```diff
@@ -0,0 +1,394 @@
+        },
+        "image": {
+            "type": "string",
+            "description": "A URI pointing to a resource with mime type image/* representing the asset to which this token represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive. If the token represents a media file of different mime-type or of higher quality defined in property 'raw_media_file_uri', then this image should be used as preview image like a cover for music, or an low-res image."
+        },
+        "attributes": {
+            "type": "array",
+            "description": "Arbitrary attributes. Values may be strings, numbers, object or arrays."
+            "items: {
+                "type": "object",
+                "required": ["value"],
+                "properties": {
+                    "display_type": "string",
+                    "trait_type": "string",
+                    "value": {"oneOf": [{"type": "object"}, {"type": "string"}, {"type": "number"}, {"type": "integer"}, {"type": "boolean"} {"type: "array"}},
+                }
+            }
+        },
+        "properties": {
+            "type": "object",
+            "description": "Arbitrary properties. Values may be strings, numbers, object or arrays."
+        },
+        "localization": {
+            "type": "object",
+            "required": ["uri", "default", "locales"],
+            "properties": {
+                "uri": {
+                    "type": "string",
+                    "description": "The URI pattern to fetch localized data from. This URI should contain the substring `{locale}` which will be replaced with the appropriate locale value before sending the request. See section about localization for more rules"
+                },
+                "default": {
+                    "type": "string",
+                    "description": "The locale of the default data within the base JSON"
+                },
+                "locales": {
+                    "type": "array",
+                    "description": "The list of locales for which data is available. These locales should conform to those defined in the Unicode Common Locale Data Repository (http://cldr.unicode.org/)."
+                }
+            }
+        },
+        "image_data": {
+            "type": "string",
+            "description": "Raw SVG image data. Deprecated. Use `properties.image_data`."
+        },
+        "external_url": {
+            "type": "string",
+            "description": "Url to view the item on a 3rd party web site. Deprecated. Use `properties.external_url`."
+        },
+        "animation_url": {
+            "type": "string",
+            "description": "URL to a multi-media attachment for the item. Deprecated. Use `properties.animation_url`."
+        },
+    }
+}
+```
+
+The lengths of string values is not restricted. Nowadays, clients should be smart enough to deal with values of different lengths.
```

Comment:
Is there a reasonable limit we can advise here?  Like, maybe no longer than 1024 characters is recommended?